utils/build-script

```diff
@@ -2018,6 +2027,11 @@ iterations with -O",
     parser.add_argument(
         "--enable-tsan-runtime",
         help="enable Thread Sanitizer on the swift runtime")
+    parser.add_argument(
+        "--enable-lsan",
+        help="enable Leaks Sanitizer for swift tools",
```

Comment:
I ran into a python lint issue, so I addressed your comment in the updated commit where I also fixed the python lint issues.